valkey.conf

```diff
@@ -387,6 +387,12 @@ databases 16
 # ASCII art logo in startup logs by setting the following option to yes.
 always-show-logo no
 
+# To prevent sensitive user information, such as PII, from being recorded in the
+# server log file, user data is hidden from the log by default. If you need to
+# log user data for debugging or troubleshooting purposes, you can disable this
+# feature by changing the config value to no.
```

Comment:
```suggestion
# User data, including keys and values, client names, and ACL usernames can be
# logged as part of assertions and other error cases. To prevent sensitive user
# information, such as PII,  from being recorded in the server log file, user data is
# hidden from the log by default. If you need to log user data for debugging or
# troubleshooting purposes, you can disable this  feature by changing the config
# value to no.
```